modules/@apostrophecms/doc-type/index.js

```diff
@@ -1502,7 +1502,7 @@ module.exports = {
           canCreate: self.apos.permission.can(req, 'create', self.name, 'draft'),
           canEdit: self.apos.permission.can(req, 'edit', self.name, 'draft'),
           canPublish: self.apos.permission.can(req, 'publish', self.name),
-          canArchive: self.apos.permission.can(req, 'archive', self.name)
+          canArchive: self.apos.permission.can(req, 'delete', self.name)
```

Comment:
I suppose we can keep the canArchive method name. But I didn't find where this one is used (there are others canArchive in the frontend but not this one)

Reply:
Agreed I don't see it used but we can have it here for bc in case someone else uses it